src/errors/test-run/templates.js

```diff
@@ -108,6 +139,30 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.selectActionElementNonEditableError]: err => markup(err, `
```

Comment:
`actionElementNonEditableError`. Try to avoid action-specific naming as much as possible, to make error messages reusable.